git/foundations_git/introduction_to_git.md

```diff
@@ -54,6 +54,6 @@ This section contains questions for you to check your understanding of this less
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-*   [Git and GitHub in plain English](https://blog.red-badger.com/blog/2016/11/29/gitgithub-in-plain-english)
+*   [Git and GitHub in plain English](https://content.red-badger.com/resources/what-is-git-and-github)
```

Comment:
I thnk we should change the link text to reflect the article heading
```suggestion
*   [What is Git and GitHub?](https://content.red-badger.com/resources/what-is-git-and-github)
```

Reply:
Indeed ! Thanks for pointing it out.